src/share/classes/com/sun/btrace/agent/TraceOutputWriter.java

```diff
@@ -149,7 +149,7 @@ private void nextWriter() {
         private FileWriter getNextWriter() throws IOException {
         	currentFileWriter.close();
         	File scriptOutputFile_renameFrom = new File(path + File.separator + baseName);
-        	File scriptOutputFile_renameTo = new File(path + File.separator + baseName.substring(0, baseName.indexOf("-")) + ".btrace." + (counter++));
+        	File scriptOutputFile_renameTo = new File(path + File.separator + baseName + "." + (counter++));
```

Comment:
Removing this for now. My automatic scripts to pick up these files do work as intended even with the dash and the timestamp. So leaving this in for now.